src/ui/LiveSplit.tsx

```diff
@@ -367,6 +359,20 @@ export class LiveSplit extends React.Component<{}, State> {
         exportFile("layout.ls1l", JSON.stringify(layout, null, 4));
     }
 
+    public loadDefaultSplits() {
+        this.setState({
+            ...this.state,
+            timer: this.getTimerForDefaultRun(),
```

Comment:
This creates a whole new timer that the hotkey system doesn't know about, so this breaks the hotkeys. Instead, we have a method on the timer that replaces the underlying run.